packages/valory/skills/olas_week_abci/prompts.py

```diff
@@ -33,7 +33,7 @@
 and so on
 
 Your response should follow the format:
-"Week [Week number] in Olas
+"Week [Week number] '23 in Olas
```

Comment:
Turns out these lines of the template and the matching line of the response are not actually used. So can just hardcode like this.

Reply:
Shouldnt we pass the year here as a variable as well? Or at least use 24

Reply:
Yes, that's right.